src/smartContract.ts

```diff
@@ -0,0 +1,446 @@
+import { keccak256 } from "@ethersproject/keccak256";
+import { toUtf8Bytes } from "@ethersproject/strings";
+
+export const ERC20Address = "0x1993e340B90eA07c9f646B60C0e9B5148253902e";
```

Comment:
while ABI is not changing the fast, this might no be the case for the address, let's add this to env vars in next PR. same for the one below